cli/spec/kontena/cli/stacks/list_command_spec.rb

```diff
@@ -60,4 +61,77 @@
       end
     end
   end
+
+  context "command output layout" do
+    before do
+      allow(client).to receive(:get).with('grids/test-grid/stacks').and_return({
```

Comment:
No need for  `({})`, can be just `('stacks' => [])`

Reply:
✅ RESOLVED